src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -12,7 +12,9 @@ internal enum ConversionKind : byte
         Identity,
         ImplicitNumeric,
         ImplicitEnumeration,
+        ImplicitTupleLiteral,
```

Comment:
no, conversions from expression are implicit only.
Since explicit conversions are super-set of implicit the following will work regardless -
``` x = (byte, byte)(1,1) ```

that'd be just explicit conversion from (int, int). Explicit conversions are NYI in this change though..